packages/rancher-windows-exporter/charts/scripts/run.ps1

```diff
@@ -36,5 +36,9 @@ function Transfer-File
     $null = Copy-Item -Force -Path $Src -Destination $Dst
 }
 
+# Copy binary
 Create-Directory -Path "c:\host\etc\windows-exporter"
-Transfer-File -Src "c:\etc\windows-exporter\windows-exporter.exe" -Dst "c:\host\etc\windows-exporter\windows-exporter.exe"
\ No newline at end of file
+Transfer-File -Src "c:\etc\windows-exporter\windows-exporter.exe" -Dst "c:\host\etc\windows-exporter\windows-exporter.exe"
+
+# Run wins
+Invoke-Expression -Command c:\entry.ps1
```

Comment:
This script is packaged directly into the windows_exporter-package image: https://github.com/rancher/windows_exporter-package/blob/master/package/windows/entry.ps1